Content.Server/GameTicking/GameTicker.GamePreset.cs

```diff
@@ -226,7 +226,7 @@ public bool OnGhostAttempt(EntityUid mindId, bool canReturnGlobal, bool viaComma
                 return false;
             }
 
-            if (HasComp<GhostComponent>(playerEntity))
+            if (TryComp<GhostComponent>(playerEntity, out var comp) && !comp.CanGhostInteract)
```

Comment:
Yeah I don't like this. The behavior here isn't obvious at all. Make a new data field instead?

Reply:
It SOUNDS unintuitive, but that's basically the "this is an admin ghost" datafield on the GhostComponent. It's the only scenario where it's used, the only thing it could mean. Should we add a redundant datafield for clarity? Or perhaps would it be better to rename it (and every reference of it) to "IsAdminGhost" or something like that?

Reply:
Ideally a new property would be best but yeah I just looked at the code and see other code is already using this as an "is admin ghost" check so...